### ISSUE TITLE
Multitasking support, native or backgrounder

### ISSUE THREAD

**Issue by GoogleCodeExporter** [StonedRightMeow17/itransmission#6]
```
What steps will reproduce the problem?
1. Closing the app, upon restart needs to resume all torrents
2.
3.

What is the expected output? What do you see instead?
To continue downloading/uploading in the background. Instead I see the app 
terminating on close

What version of the product are you using? On what operating system?
.4, iOS 4.3.3, iPhone 4, downloaded from this website on my iDevise using 
safari download manager and installed with installous.

Please provide any additional information below.
Not even just expected multitasking doesn't work, it doesn't seem to react to 
backgrounder either

```

Original issue reported on code.google.com by `giulianohicks@gmail.com` on 2 Jun 2011 at 5:33

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
One can get around this by editing "info.plist" in 
"/Applications/iTransmission". 
Once the file is open, change "UIApplicationExitsOnSuspend" to false. This will 
suspend it when in the background. Once this is done you can use Backgrounder 
to have your torrents run in the background. 
```

Original comment by `3radicat...@gmail.com` on 13 Jun 2011 at 3:26

**Comment by GoogleCodeExporter** [comment]
```
You will have to respring after editing the plist file. 
```

Original comment by `3radicat...@gmail.com` on 13 Jun 2011 at 3:37

**Comment by GoogleCodeExporter** [comment]
```
Works! Great workaround thanks! Now this app is one step closer to perfection. 
The next thing needed is the ability to select files from within the torrent. 

Thanks again
```

Original comment by `protyush...@gmail.com` on 13 Jun 2011 at 8:04

**Comment by GoogleCodeExporter** [comment]
```
It worked for me too. Thank you!
```

Original comment by `jajuanga...@gmail.com` on 14 Jun 2011 at 10:50

**Comment by GoogleCodeExporter** [comment]
```
I did not have to edit any plist file in order to make iTransmission continue 
running when switching to another app. I simply went into Backgrounder, went 
into Overrides, clicked "Add" and chose iTransmission, then set the following 
settings:

Backgrounding method: Auto Detect(Native if supported, else Backgrounder method 
will be used.)

Backgrounding state: Enable at Launch ON, Stay Enabled ON

Minimize on Toggle: ON

Boom! iTransmission stays running, downloading torrents, until I manually quit 
the app.

This was done an AT&T (GSM) iPhone 4 running iOS 4.3.2
```

Original comment by `hfe...@gmail.com` on 18 Jun 2011 at 8:18

**Comment by GoogleCodeExporter** [comment]
```
Or jus dl the app from my link and its already done for u ;)
```

Original comment by `machster...@hotmail.com` on 18 Jun 2011 at 8:29

**Comment by GoogleCodeExporter** [comment]
```
@hfelsh offers a solution. I'm closing this issue. 
```

Original comment by `ccp0...@gmail.com` on 19 Jun 2011 at 1:32
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Issue 10 has been merged into this issue.
```

Original comment by `ccp0...@gmail.com` on 19 Jun 2011 at 1:44

**Comment by GoogleCodeExporter** [comment]
```
@Mike Chen I hope you reconsider closing this issue. Using Backgrounder is 
hardly a solution. The app should support native multitasking that it doesn't. 
As of now, editting the plist is the beat option. I hope u include that in 
future releases
```

Original comment by `protyush...@gmail.com` on 19 Jun 2011 at 4:59

**Comment by GoogleCodeExporter** [comment]
```
@protyush native multitasking doesn't allow me to do persistent backgrounding. 
Background is the only solution available. 
```

Original comment by `ccp0...@gmail.com` on 19 Jun 2011 at 5:47

**Comment by GoogleCodeExporter** [comment]
```
sorry for typo. Backgrounder is the only solution. 
```

Original comment by `ccp0...@gmail.com` on 19 Jun 2011 at 5:47

**Comment by GoogleCodeExporter** [comment]
```
Umm backgrounder is not needed... Plist edit works perfect by itself
```

Original comment by `machster...@hotmail.com` on 19 Jun 2011 at 11:06

**Comment by GoogleCodeExporter** [comment]
```
And its definately "not" the only solution....
```

Original comment by `machster...@hotmail.com` on 19 Jun 2011 at 11:10

**Comment by GoogleCodeExporter** [comment]
```
Backgrounder does not work on IOS 6, please reopen this issue.
```

Original comment by `dbo...@gmail.com` on 9 Apr 2013 at 5:54
